extensions/wikia/AdEngine/js/template/floating-rail.js

```diff
@@ -0,0 +1,101 @@
+/*global define*/
+define('ext.wikia.adEngine.template.floating-rail', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.uapContext',
+	'ext.wikia.adEngine.utils.math',
+	'ext.wikia.adEngine.adHelper',
+	'jquery',
+	'wikia.log',
+	'wikia.document',
+	'wikia.window'
+], function (adContext, uapContext, math, adHelper, $, log, doc, win) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has 8 parameters which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)